css/main.css

```diff
@@ -105,8 +105,11 @@ body {
 
 .logo-img {
   width: 200px;
+  height: 200px;
   display: block;
   margin: 20px auto;
+  overflow-anchor: none;
```

Comment:
I don't think we need this, check what changes are there when this is not applied

Reply:
Yes, removed and checked